wikimedia/wikipedia-ios
# Bug fix - Native Editor last character bold/italic state

**Phabricator:** 
N/A

### Notes
This fixes a bug I noticed where the bold / italic "selected" button state didn't show when the cursor was directly before the trailing syntax.

### Test Steps
In Staging scheme, run app, visit an article to edit. Go to an area with bold or italic syntax highlighting. Confirm when putting the cursor directly before the trailing formatter, the selected state appears on the button menus. Confirm you can toggle the formatter on and off as expected.

### Screenshots/Videos

Before:
![IMG_0284](https://github.com/wikimedia/wikipedia-ios/assets/3620196/0fbe3553-73da-437a-ab4d-36a8051a56ca)

After:
![IMG_0286](https://github.com/wikimedia/wikipedia-ios/assets/3620196/c5ac7b4c-016f-407e-a31a-a7013067560c)
